### DISCUSSION TITLE
Where are the executable files?

### DISCUSSION THREAD

**Discussion by djeaton** [receyuki/stable-diffusion-prompt-reader#44] [Q&A] [answered]
I downloaded and extracted the zip file because I really could use this, but can't find a bat file or exe file to execute.  Am I missing something?  I see a "setup.py" file, but I don't know python and don't know what to do with that.

**Comment by receyuki** [comment, marked as answer]
https://github.com/receyuki/stable-diffusion-prompt-reader/releases/latest

Download and unzip SD-Prompt-Reader-X.X.X-Windows-x64.zip

**Reply by djeaton** [reply]
Thanks.  I'm totally unexperienced with git and github and probably just downloaded the source code or something.

**Reply by receyuki** [reply]
No worries. Usually, if there's an executable version, it will be in the releases. If it's not in the releases, there should be instructions in the readme about how to use it.
